src/main/java/com/laytonsmith/core/Static.java

```diff
@@ -795,8 +550,8 @@ public static UUID GetUUID(Mixed subject, Target t) {
 		return GetUUID(subject.val(), t);
 	}
 
-	public static MCOfflinePlayer GetUser(Mixed search, Target t) {
-		return GetUser(search.val(), t);
+	public static MCOfflinePlayer GetUser(Mixed search, Target t, Environment env) {
```

Comment:
These two GetUser() methods are probably used by extensions, but I can't think of one off the top of my head.